sdk/communication/azure-communication-identity/src/test/java/com/azure/communication/identity/TokenCustomExpirationTimeHelper.java

```diff
@@ -0,0 +1,27 @@
+package com.azure.communication.identity;
+
+import org.junit.jupiter.params.provider.Arguments;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class TokenCustomExpirationTimeHelper {
+
+    static Stream<Arguments> getValidExpirationTimes() {
+        List<Arguments> argumentsList = new ArrayList<>();
```

Comment:
have to check what if happens if you pass null

Reply:
Added case for null.